.circleci/config.yml

```diff
@@ -22,10 +22,10 @@ aliases:
     - run:
         name: "Import signing key"
         command: |
+          gpg --keyserver keyserver.ubuntu.com \
```

Comment:
We're specifying the Ubuntu keyserver here to mirror the [approach taken](https://github.com/jodersky/sbt-gpg/blob/master/.ci/build) by @jodersky. We've found that this keyserver was updated more quickly than the default.

Reply:
(sorry for the needless ping Jakob, I'm way too used to at-mentioning all over the place, thanks for the inspiration!)